website/src/components/NavBar.js

```diff
@@ -63,6 +111,27 @@ export default function ButtonAppBar() {
                     { loggedIn ?
                     (
                         <>
+                            {isAdmin &&
+                                <Box>
+                                    <FormControl size='small' sx={{m: 1, minWidth:100}} variant={"filled"}>
+                                        <InputLabel id='student-dropdown-label'>Student</InputLabel>
+                                        <Select
+                                            labelId='student-dropdown-label'
+                                            id='student-dropdown'
+                                            label='student'
+                                            onChange={loadStudentData}
+                                            style={{backgroundColor: "white"}}
+                                            defaultValue={selectedStudent}
+                                        >
+                                            {
+                                                students.map((student) => (
+                                                    <MenuItem key={student[1]} value={student[1]}>{student[0]}</MenuItem>
+                                                ))
```

Comment:
My IntelliSense on WebStorm flags the semicolon because its in a lambda function

Reply:
I think it might be because this is in the body of the html render and the way that intellisense handles this area of JS is diff from the main body of JS for whatever reason.  Tbh though, no red squiggle >> red squiggle for this area since the logic shouldn't be rendering more than one "fragment".